src/lib/components/SearchBar/SearchBar.js

```diff
@@ -31,33 +31,63 @@ class SearchBar extends Component {
     this.updateQueryString(this.state.currentValue);
   };
 
+  onBtnSearchClick = (event, input) => {
```

Comment:
I am not sure that you have to make these 2 methods overridable via props. If it is because it is used in the homepage of ILS, then the cmp in the homepage should be removed and replaced with a simple fake search input.
Is it needed for other use cases?
If I remember correctly, you only need to make the "style" configurable via prop.